.github/workflows/auto-testing.yml

```diff
@@ -24,14 +24,18 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         if [ -f requirements-dev.txt ]; then python -m pip install -r requirements-dev.txt; fi
+    - name: Ensure generate-code.py doesn't throw any error
+      run: |
+        python generate-code.py
+    - name: Update version in linebot/__about__.py
+      run: |
+        sed -i 's/__version__ = "__LINE_BOT_SDK_PYTHON_VERSION__"/__version__ = "12.34.5"/g' linebot/__about__.py
```

Comment:
What's `12.34.5`?

Reply:
I should have written why this uses `12.34.5`...
Tox (test library) rejects running test if version is illegal. As a workaround, this embeds dummy version.